README.md

```diff
@@ -13,7 +13,9 @@ The apps that you will be creating in Mod 2 will make heavy use of these four ac
 
 Let's use Rails to build an application where we can manage some tasks.
 
-We're going to follow the MVC design pattern, which Rails uses by default, to implement the CRUD actions for our Task Manager app. For a quick explanation of MVC, please read through [this](https://medium.freecodecamp.org/simplified-explanation-to-mvc-5d307796df30) post.
+TODO: Add an image very similar to this but without the view portion: https://backend.turing.edu/module2/lessons/images/mvc_rails.png
```

Comment:
This image is now updated (same name) so you can use it here if you'd like. The MVC lesson is now updated as well. I think the original line 16 works fine. MVC is a design pattern and a view layer still exists even if it's not HTML. We could link to our own Intro to MVC lesson instead of free code camp if we prefer to direct them to a resource that will talk about Views as presentation of data rather than specifically HTML renderings.

Reply:
Oh great, thanks Erin! I'll just go ahead and use that same image and a link to that lesson for further pre-learning!